spec/Records.tex

```diff
@@ -161,9 +159,9 @@ \section{Record Variable Declarations}
 
 In the context of a function body, a record variable declaration
 causes storage to be allocated sufficient to store all of the fields in that
-record type.  The record variable is initialized through a call to its
-default initializer.  The default initializer for a record is defined in the
-same way as the default initializer for a class (\rsec{Default_Initialization}).
+record type.  The record variable is initialized with a call to an
+initializer that accepts zero actual arguments. If multiple such initializers
+exist, an ambiguity error is raised.
```

Comment:
I would add an example of a case where ambiguity occurs, to make it obvious

Reply:
How about I remove the line about ambiguity errors, since the Classes section says it behaves like normal function resolution?

Reply:
That works for me